drivers/include/periph/usbdev.h

```diff
@@ -310,6 +310,15 @@ typedef struct usbdev_driver {
      */
     void (*esr)(usbdev_t *dev);
 
+    /**
+     * @brief Stall both OUT and IN packets on endpoint 0 until a setup packet
+     * is received.
+     *
+     * @note The stall condition is automatically cleared after receiving a
+     * setup packet.
+     */
```

Comment:
Do you mean by hardware or software ?
IIRC, the ST USB IP that I am working on, will rely on software to do this action.

Reply:
> Do you mean by hardware or software ?

Yes. Either by software (the usbdev implementation) or by hardware. USBUS assumes that it can stall endpoint 0 but that it doesn't have to clear the stall condition to receive the new setup request. [MbedOS has a similar assumption](https://os.mbed.com/docs/mbed-os/v6.9/porting/usb-device.html) for their USBDevice implementations: 
> Endpoint 0 stall automatically clears on reception of a setup packet.

Reply:
Do you mind rewording this a bit ?
Something like
`@note The stall condition should be cleared automatically either by hardware or by the usbdev implementation after receiving a setup packet.`

Reply:
rephrased!